2010.04.15
处理第一次成功调用远程对象，第二次无法调用问题分析：
1.注册的监听器还在运行，因为当调用时仍然可以使用监听器，只是无法取到值。
2.远程实例存在于静态内存中，第二次调用可能有问题。
3.本地只实例化一次，可能存在问题。

2010.04.22
完成用户管理的数据呈现，存在问题是编码有冗余的地方，以及需要总结设计过程。